editor.planx.uk/src/@planx/components/FileUploadAndLabel/Public.tsx

```diff
@@ -136,13 +136,11 @@ function Component(props: Props) {
       <QuestionHeader {...props} />
       <DropzoneContainer>
         <FileStatus status={fileUploadStatus} />
-        <ErrorWrapper error={validationError} id={props.id}>
-          <Dropzone
-            slots={slots}
-            setSlots={setSlots}
-            setFileUploadStatus={setFileUploadStatus}
-          />
-        </ErrorWrapper>
+        <Dropzone
+          slots={slots}
+          setSlots={setSlots}
+          setFileUploadStatus={setFileUploadStatus}
+        />
```

Comment:
nit: I actually much prefer the ErrorWrapper wrapping the Dropzone in the particular case of trying to continue before uploading anything because it draws attention to the action needed to proceed. Felt like it aligned well to the govuk guidance here: https://design-system.service.gov.uk/components/file-upload/#error-messages

The floating "Upload at least one file" looks a little odd to me, and if we keep this, I think would technically be a better fit for the govuk [error summary fully-bordered style](https://design-system.service.gov.uk/components/error-summary/) because it's not wrapping any particular element on the page?

But I guess this approach let's you avoid multiple ErrorWrappers that conditionally render based on the specific `validationError` message?